src/Common/src/Interop/Unix/libc/Interop.gethostname.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Yeah, this one is stale I think. There's a HostInformationPal class that we use now that was implemented recently. Will remove this.


On Oct 7, 2015, at 5:46 PM, Stephen Toub <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/Common/src/Interop/Unix/libc/Interop.gethostname.cs<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fdotnet%2fcorefx%2fpull%2f3707%23discussion_r41466066&data=01%7c01%7cerme%40microsoft.com%7c9d060c96a32e4ea2547308d2cf79de63%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=HxETZlPNWU5mvnKH799kI4vlaf98HOvzb4xo%2fliSQoE%3d>:

> @@ -0,0 +1,38 @@
> +// Copyright (c) Microsoft. All rights reserved.


This is also stale, right? This was already moved to the shim?

-
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fdotnet%2fcorefx%2fpull%2f3707%2ffiles%23r41466066&data=01%7c01%7cerme%40microsoft.com%7c9d060c96a32e4ea2547308d2cf79de63%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=DIOFq465DBESdg9dN9gBVDfQhVxca2pfhRC6qE87pp0%3d>.